datasets/titanic/titanic_by_openml_converter.json

```diff
@@ -1,124 +1,121 @@
+          "source": "#{Titanic_arff/pclass}"
         },
         {
+          "name": "survived",
           "@type": "ml:Field",
           "dataType": "Boolean",
-          "name": "survived",
-          "source": "#{Titanic (parquet)/survived}"
+          "source": "#{Titanic_arff/survived}"
         },
         {
+          "name": "name",
           "@type": "ml:Field",
           "dataType": "Text",
-          "name": "name",
-          "source": "#{Titanic (parquet)/name}"
+          "source": "#{Titanic_arff/name}"
         },
         {
+          "name": "sex",
           "@type": "ml:Field",
           "dataType": "Text",
-          "name": "sex",
-          "source": "#{Titanic (parquet)/sex}"
+          "source": "#{Titanic_arff/sex}"
         },
         {
+          "name": "age",
           "@type": "ml:Field",
           "dataType": "Number",
-          "name": "age",
-          "source": "#{Titanic (parquet)/age}"
+          "source": "#{Titanic_arff/age}"
         },
         {
+          "name": "sibsp",
           "@type": "ml:Field",
           "dataType": "Number",
-          "name": "sibsp",
-          "source": "#{Titanic (parquet)/sibsp}"
+          "source": "#{Titanic_arff/sibsp}"
         },
         {
+          "name": "parch",
           "@type": "ml:Field",
           "dataType": "Number",
-          "name": "parch",
-          "source": "#{Titanic (parquet)/parch}"
+          "source": "#{Titanic_arff/parch}"
         },
         {
+          "name": "ticket",
           "@type": "ml:Field",
           "dataType": "Text",
-          "name": "ticket",
-          "source": "#{Titanic (parquet)/ticket}"
+          "source": "#{Titanic_arff/ticket}"
         },
         {
+          "name": "fare",
           "@type": "ml:Field",
           "dataType": "Number",
-          "name": "fare",
-          "source": "#{Titanic (parquet)/fare}"
+          "source": "#{Titanic_arff/fare}"
         },
         {
+          "name": "cabin",
           "@type": "ml:Field",
           "dataType": "Text",
-          "name": "cabin",
-          "source": "#{Titanic (parquet)/cabin}"
+          "source": "#{Titanic_arff/cabin}"
         },
         {
+          "name": "embarked",
           "@type": "ml:Field",
           "dataType": "Text",
-          "name": "embarked",
-          "source": "#{Titanic (parquet)/embarked}"
+          "source": "#{Titanic_arff/embarked}"
         },
         {
+          "name": "boat",
           "@type": "ml:Field",
           "dataType": "Text",
-          "name": "boat",
-          "source": "#{Titanic (parquet)/boat}"
+          "source": "#{Titanic_arff/boat}"
         },
         {
+          "name": "body",
           "@type": "ml:Field",
           "dataType": "Number",
-          "name": "body",
-          "source": "#{Titanic (parquet)/body}"
+          "source": "#{Titanic_arff/body}"
         },
         {
+          "name": "home.dest",
```

Comment:
Nit: Why a "."  in the name?

Reply:
As name I used the exact name of this attribute/column: `@attribute 'home.dest' string`
Should I clean up these values (remove capitals and special characters)? 
If so, where do we put the literal name of the attribute/column?